### PR TITLE
Revise Deploy script to also be interactive and easy to use!

### PR DESCRIPTION
**Pull request by paolopaolopaolo**
@rmutter, @baylee: Tiny feature that wraps the ansible-playbook call in an easy/interactive menu for deployment. It looks inside the `orchestration/inventory` folder for potential environments.

image

Just need some review before I merge this sucker in. I've also been updating the field guide with all this goodness.

### CODE REVIEW

File: fabfile/deploy.py [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
-        with in_project():
-            with settings(warn_only=True):
-                local('ansible-playbook -i orchestration/inventory/{} orchestration/site.yml'.format(environment))
+def deploy():
+    """
+    Starts deploy process
+    :return:
+    """
+    ENVIRONMENT_OPTIONS = {}
+    print "Option\tInventory"
```
Comment (rmutter) [inline fabfile/deploy.py]: let's use `print()` throughout yurt - probably will want to switch it to python 3 when our dependencies support it

**Comment by rmutter** [comment]
@paolopaolopaolo this is awesome functionality and will be super helpful for people less familiar with cli/yurt! It would also be great, if we could pass a flag in to force select an option.

For example either `fab deploy --env=development` or `fab deploy development`. This would make it easier for us more familiar with it to rerun it quickly. And if you don't pass in that optional argument it will give you the interactive prompt.

**Comment by paolopaolopaolo** [comment]
@rmutter made the fixes that were recommended. Let me know if I can merge!

**Comment by rmutter** [comment]
🎉

**Pull request merged**